lv_conf_template.h

```diff
@@ -137,6 +137,12 @@
  * - Rendering engine needs to support 3x3 matrix transformations. */
 #define LV_DRAW_TRANSFORM_USE_MATRIX            0
 
+/** Selectively disable lv_draw_sw_blend function and use a custom one. Possible options:
+ * - LV_DRAW_BLEND_CUSTOM_NONE
+ * - LV_DRAW_BLEND_CUSTOM_NEMA_GFX */
+
+#define LV_DRAW_BLEND_CUSTOM LV_DRAW_BLEND_CUSTOM_NONE
```

Comment:
As it's realted to SW render please move it to the other LV_DRAW_SW_ config option and call it `LV_DRAW_SW_BLEND_CUSTOM`